bookmark-it/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "manifest_version": 2,
+  "name": "Bookmark it!",
+  "version": "1.0",
+  "description": "A simple bookmark button",
+  "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/bookmark-it",
+  "icons": {
+    "48": "icons/bookmark-it.png",
+    "96": "icons/bookmark-it@2x.png"
+  },
+
+  "manifest_version": 2,
+  "name": "bookmark-it",
+  "version": "1.0",
+
+  "applications": {
+    "gecko": {
+      "id": "bookmark-it-extension@mozilla.org"
```

Comment:
This should have strict_min_version. If search() only exists in Nightlies, I think we could set it to "47.0a1".